src/bundle/Resources/public/js/CKEditor/custom-tags/block-custom-tag/custom-tag-editing.js

```diff
@@ -31,11 +31,35 @@ class IbexaCustomTagEditing extends Plugin {
         conversion.for('editingDowncast').elementToElement({
             model: 'customTag',
             view: (modelElement, { writer: downcastWriter }) => {
+                const customTagName = modelElement.getAttribute('customTagName');
                 const container = downcastWriter.createContainerElement('div', {
                     'data-ezelement': 'eztemplate',
-                    'data-ezname': modelElement.getAttribute('customTagName'),
-                    class: 'ez-custom-tag',
+                    'data-ezname': customTagName,
+                    class: 'ibexa-custom-tag',
                 });
+                const header = downcastWriter.createUIElement('div', { class: 'ibexa-custom-tag__header' }, function(domDocument) {
```

Comment:
why not arrow function?

Reply:
With arrow function this is lost and CKEditor throws an error.